docs/source/user/installing.rst

```diff
@@ -5,7 +5,8 @@ Installation
 Compatibility
 -------------
 
-Numba is compatible with Python 3.7 or later, and Numpy versions 1.17 or later.
+Numba is compatible with Python 3.7 or later, and Numpy versions from 1.17 up
```

Comment:
```suggestion
Numba is compatible with Python 3.7 or later, and NumPy versions from 1.17 up
```
though not necessary to get this merged!

Reply:
fixed in abc4f8d62e94b54221933fc5149aef3a508ede93